sympy/core/compatibility.py

```diff
@@ -38,12 +38,12 @@
 
 Moved modules:
     * `reduce()`
-    * `StringIO()`
-    * `cStringIO()` (same as `StingIO()` in Python 3)
+    * `cStringIO()` (uses `StringIO.cStringIO` in Python 2, same as `StingIO()`
+      in Python 3, use `io.StringIO` for normal `StringIO`)
     * Python 2 `__builtins__`, access with Python 3 name, `builtins`
 
 Iterator/list changes:
-    * `xrange` removed in Python 3, import `xrange` for Python 2/3 compatible
```

Comment:
Grepping for xrange only shows a handful of places, and none of them are imports from `compatiblity.py`, it used to be like that but it was changed in 70b2ce9d348567aff1ffcd18475d752291519b95